src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/courtsentencing/CourtSentencingMigrationService.kt

```diff
@@ -67,18 +67,38 @@ class CourtSentencingMigrationService(
           courtCaseId = nomisCaseId,
         )
 
-      courtSentencingDpsService.createCourtCaseMigration(nomisCourtCase.toMigrationDpsCourtCase()).also { dpsCourtCaseCreateResponse ->
-        createCourtCaseMapping(nomisCourtCase = nomisCourtCase, dpsCourtCaseCreateResponse = dpsCourtCaseCreateResponse, context)
+      if (nomisCourtCase.combinedCaseId != null) {
+        log.info("Will not migrate the court case since it is the locked down part of a linked case, NOMIS caseId is $nomisCaseId")
         telemetryClient.trackEvent(
-          "court-sentencing-migration-entity-migrated",
+          "court-sentencing-migration-entity-skipped",
```

Comment:
I suppose it doesn't matter but interestingly I would of naturally filtered out in nomis API for get IDS - but I guess it doesn't matter - be aware - your numbers of estimate cases won't add up if that matters in the dashboard

Reply:
yes true but it is a temporary measure and can the numbers can be easily checked and compared by the custom events